src/main/java/com/conveyal/datatools/manager/jobs/PublishProjectFeedsJob.java

```diff
@@ -62,7 +68,12 @@ public void jobLogic () {
                     }
                     else {
                         // ensure latest feed is written to the s3 public folder
-                        fs.makePublic();
+                        try {
+                            fs.makePublic();
+                        } catch (Exception e) {
+                            status.fail("Failed to make s3 folder public", e);
```

Comment:
This failure message doesn't seem super accurate/helpful. Perhaps `Failed to make GTFS files public on S3`?